pages/api/video.ts

```diff
@@ -0,0 +1,118 @@
+import fs from "fs";
+import os from "os";
+import path from "path";
+import { PaintPoint, ProtocolClient } from "@replayio/protocol";
+import { NextApiRequest, NextApiResponse } from "next";
+
+import { createClient } from "../../src/api/socket";
+import { defer } from "../../src/api/utils";
+
+const ffmpegPath = require("@ffmpeg-installer/ffmpeg").path;
+
+const ffmpeg = require("fluent-ffmpeg");
+ffmpeg.setFfmpegPath(ffmpegPath);
+
+async function fetchPaintPoints({
+  client,
+  sessionId,
+}: {
+  client: ProtocolClient;
+  sessionId: string;
+}) {
+  const findPaints = client.Graphics.findPaints({}, sessionId);
+  let gPaints: PaintPoint[] = [];
+  client.Graphics.addPaintPointsListener(({ paints }) => {
+    gPaints.push(...paints);
+  });
+
+  await findPaints;
+  console.log("found points", gPaints.length);
```

Comment:
I assume this (and the other console logs) were intentionally left in place?

Reply:
Yes - on the backend that's the only way to see what happened